jaseci_core/jaseci/utils/id_list.py

```diff
@@ -27,7 +27,9 @@ def __init__(self, parent_obj, auto_save=True, in_list=None):
     def cache_reset(self):
         self.cached_objects = []
 
-    def add_obj(self, obj, push_front=False, allow_dups=False, silent=False):
+    def add_obj(
+        self, obj, push_front=False, allow_dups=False, silent=False, bypass=False
+    ):
```

Comment:
@amadolid I talked with @marsninja about this PR. He said that if we set `auto_save` as false for the fast_edge_id IdList should have the same effect as what we are trying to do with bypass. Can you test that out and make the change if it works?

Reply:
I think, I implement this before and saw that it might not trigger the saving on other approach reason for just bypassing only the buildup part

we call `smart_add_edge` on edge.connect and this function will try to add to _fast_edge_ids. I'm assuming it supposed to do the saving on parent's node whenever we trigger the smart_add_edge. `auto_save` false will disable this
```python
def smart_add_edge(self, obj):
        # make sure fast edges built
        if not len(self._fast_edge_ids):
            self.smart_build_fast_edge_ids()
        # add new edge
        self._fast_edge_ids.add_obj(obj)
        # then store how needed
        if obj.is_fast():
            self.smart_edge_to_fast_edge(obj)
        else:
            self.edge_ids.add_obj(obj)
```